tests/foreman/api/test_discoveredhost.py

```diff
@@ -294,8 +311,18 @@ def test_positive_auto_provision_pxe_host(self):
 
```

Comment:
`:CaseAutomation: Automated`  can be removed for both tests as already defined on the module level

Reply:
Yeah will update that